SessionPlayer.js

```diff
@@ -0,0 +1,198 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { View, StyleSheet } from 'react-native';
+
+import PlayerActionButton from 'PlayerActionButton';
+
+/*
+ * Guides
+ */
+import GuideBreathingNormal from 'Guides/Breathing/Normal';
+import GuideBreathingPause from 'Guides/Breathing/Pause';
+import GuideBreathingFinished from 'Guides/Breathing/Finished';
+
+import GuidePowerPoseSuperHuman from 'Guides/PowerPose/SuperHuman';
+import GuidePowerPosePause from 'Guides/PowerPose/Pause';
+import GuidePowerPoseFinished from 'Guides/PowerPose/Finished';
+
+const guides = {
```

Comment:
I like this idea, but I don't like the inconsistency with naming. Can we figure out a way to not have `Normal` and `SuperHuman`? I get what you are trying to do further in the code, but naming here just doesn't work for me.